scripts/armstrong.py

```diff
@@ -0,0 +1,16 @@
+# armstrong number: 153 = 1^3+5^3+3^3
+
+def is_armstrong(n):
+    s=0
```

Comment:
We can give it a more intuitive variable name such as `sum_of_cubes` or maybe just add a comment on the side as follows:

``` python
s = 0 # sum of cubes
````

Reply:
Implemented